src/input-nested-select/types.ts

```diff
@@ -0,0 +1,57 @@
+import {
+    InputSelectOptionsProps,
+    InputSelectSharedProps,
+} from "../input-select";
+import { DropdownStyleProps, Mode } from "../shared/nested-dropdown-list/types";
+
+interface InputNestedSelectOptionsProps<V1, V2, V3>
+    extends Omit<InputSelectOptionsProps<V1>, "options"> {
+    options: L1OptionProps<V1, V2, V3>[];
+}
+
+// =============================================================================
+// INPUT SELECT PROPS
+// =============================================================================
+export interface InputNestedSelectProps<V1, V2, V3>
+    extends React.HTMLAttributes<HTMLElement>,
+        InputNestedSelectOptionsProps<V1, V2, V3>,
+        Omit<InputSelectSharedProps<V1>, "options">,
+        DropdownStyleProps {
+    readOnly?: boolean | undefined;
+    /** Behavior of dropdown list items once it is opened */
+    mode?: Mode;
```

Comment:
```suggestion
    mode?: Mode | undefined;
```

Reply:
thanks for updating nested-dropdown-list but actually input-nested-select has this issue too